File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0103U13GrdSearchOrderListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
    public class OCS0103U13GrdSearchOrderListInfo
    {
        private String _slipCode;
        private String _slipName;
        private String _hangmogCode;
        private String _hangmogName;
        private String _hospCode;
        private String _trialFlg;

        public String SlipCode
        {
            get { return this._slipCode; }
            set { this._slipCode = value; }
        }

        public String SlipName
        {
            get { return this._slipName; }
            set { this._slipName = value; }
        }

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String HangmogName
        {
            get { return this._hangmogName; }
            set { this._hangmogName = value; }
        }

        public String HospCode
        {
            get { return this._hospCode; }
            set { this._hospCode = value; }
        }

        public String TrialFlg
        {
            get { return this._trialFlg; }
            set { this._trialFlg = value; }
        }

        public OCS0103U13GrdSearchOrderListInfo() { }

        public OCS0103U13GrdSearchOrderListInfo(String slipCode, String slipName, String hangmogCode, String hangmogName, String hospCode, String trialFlg)
        {
            this._slipCode = slipCode;
            this._slipName = slipName;
            this._hangmogCode = hangmogCode;
            this._hangmogName = hangmogName;
            this._hospCode = hospCode;
            this._trialFlg = trialFlg;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0103U13GrdSearchOrderListInfo.java
package nta.med.data.model.ihis.ocsa;

public class OCS0103U13GrdSearchOrderListInfo {
	private String slipCode  ;
	private String slipName  ;
	private String hangmogCode ;
	private String hangmogName ;
	private String hospCode   ;
	private String trialFlg  ;
	
	public OCS0103U13GrdSearchOrderListInfo(String slipCode, String slipName,
			String hangmogCode, String hangmogName, String hospCode,
			String trialFlg) {
		super();
		this.slipCode = slipCode;
		this.slipName = slipName;
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
		this.hospCode = hospCode;
		this.trialFlg = trialFlg;
	}

	public String getSlipCode() {
		return slipCode;
	}

	public void setSlipCode(String slipCode) {
		this.slipCode = slipCode;
	}

	public String getSlipName() {
		return slipName;
	}

	public void setSlipName(String slipName) {
		this.slipName = slipName;
	}

	public String getHangmogCode() {
		return hangmogCode;
	}

	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}

	public String getHangmogName() {
		return hangmogName;
	}

	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}

	public String getHospCode() {
		return hospCode;
	}

	public void setHospCode(String hospCode) {
		this.hospCode = hospCode;
	}

	public String getTrialFlg() {
		return trialFlg;
	}

	public void setTrialFlg(String trialFlg) {
		this.trialFlg = trialFlg;
	}
	
}
